src/Form/__tests__/FieldCheckbox-test.js

```diff
@@ -95,33 +97,200 @@ describe('FieldCheckbox', function () {
+            {name: 'quis', checked: false, indeterminate: true}
+          ]}
+          labelClass="foo" />
+      );
+      var children = TestUtils.scryRenderedComponentsWithType(
+        instance,
+        IconCheckbox
+      );
+
+      expect(children[1].props.labelClass).toEqual('bar');
+    });
+
+    it('should call handler with \'multipleChange\' for items', function () {
+      TestUtils.Simulate.change(
+        this.inputChildren[0],
+        {target: {checked: true}}
+      );
+      var args = this.handleEventSpy.mostRecentCall.args;
+      expect(args[0]).toEqual('multipleChange');
+      expect(args[1]).toEqual('foo');
+      expect(args[2]).toEqual([{name: 'foo', checked: true}]);
+    });
+
+    it('should call handleChange with \'change\' on single item', function () {
+      var handleEventSpy = jasmine.createSpy('handleEvent');
+      var instance = TestUtils.renderIntoDocument(
+        <FieldCheckbox
+          name="foo"
+          handleEvent={handleEventSpy}
+          startValue={{name: 'bar'}}
+          labelClass="foo" />
+      );
+      var input = TestUtils.findRenderedDOMComponentWithTag(instance, 'input');
+      TestUtils.Simulate.change(input, {target: {checked: true}});
+      var args = handleEventSpy.mostRecentCall.args;
+      expect(args[0]).toEqual('change');
+      expect(args[1]).toEqual('foo');
+      expect(args[2]).toEqual({name: 'bar', checked: true});
+    });
+
+    it('should display the checked of each item', function () {
+      expect(this.children[0].props.checked).toEqual(undefined);
+      expect(this.children[1].props.checked).toEqual(true);
+      expect(this.children[2].props.checked).toEqual(false);
+    });
+
+    it('should change value of only the checked item', function () {
+      this.instance.handleChange('multipleChange', 'quis', {target: {checked: true}})
+      expect(this.handleEventSpy).toHaveBeenCalledWith(
+        // Event name
+        'multipleChange',
+        // Field name
+        'foo',
+         // Changed radio buttons
+        [{name: 'quis', checked: true}],
+         // Event fired
+        {target: {checked: true}}
+      );
+    });
+
+    it('should display the indeterminate of each item', function () {
+      expect(this.children[0].props.indeterminate).toEqual(false);
+      expect(this.children[1].props.indeterminate).toEqual(undefined);
+      expect(this.children[2].props.indeterminate).toEqual(true);
+    });
+
+  });
+
+  describe('#render', function () {
+
+    it('is not checked by default', function () {
+      var instance = instance = TestUtils.renderIntoDocument(
+        <FieldCheckbox
+          name="foo" />
+      );
+
+      var inputComponent = TestUtils.findRenderedDOMComponentWithTag(
+        instance, 'input'
+      );
+      var el = ReactDOM.findDOMNode(inputComponent);
+
+      expect(el.checked).toEqual(false);
+    });
+
+    it('is not checked by default', function () {
```

Comment:
This description should be something like `allows setting initial checked value`